.github/workflows/weekly-pulumi-update.yml

```diff
@@ -0,0 +1,89 @@
+name: weekly-pulumi-update
+on:
+  schedule:
+  - cron: 0 11 * * 1
```

Comment:
Every monday at 11AM UTC

Reply:
If you need to explain it, why not put it in the code.
```suggestion
  # Every Monday at 11AM UTC
  - cron: 0 11 * * 1
```